types/entity.go

```diff
@@ -11,6 +11,7 @@ type Entity struct {
 	UID        EntityUID    `json:"uid"`
 	Parents    EntityUIDSet `json:"parents"`
 	Attributes Record       `json:"attrs"`
+	Tags       Record       `json:"tags,omitempty"`
```

Comment:
You determined that `omitempty` is useless here, right?

Reply:
Yes, removed here: https://github.com/cedar-policy/cedar-go/pull/77/commits/9074d5b00f44be4346c13ac8b7270a09b06c541a.